sktime/utils/plotting.py

```diff
@@ -91,6 +93,25 @@ def plot_series(*series, labels=None, markers=None):
 
         plot_func(x=x, y=y, ax=ax, marker=marker, label=label, color=color)
 
+    # plot prediction intervals if present
+    if pred_int is not None:
+        # check same conditions as for earlier indices
+        if not type(index) is type(pred_int.index):  # noqa
+            raise TypeError("Found series with different index types.")
+        elif all([x in index for x in pred_int.index]):
+            ax.fill_between(
+                ax.get_lines()[-1].get_xdata(),
+                pred_int.lower,
+                pred_int.upper,
+                alpha=0.3,
+                color=ax.get_lines()[-1].get_c(),
+                label="prediction intervals",
+            )
+        else:
+            raise ValueError(
+                "pred_int has index values for which no other index values supplied"
```

Comment:
Could we also improve the error message, I guess we want to say that pred_int index does not match the index of any of the passed series.